test/reporters/verbose.js

```diff
@@ -89,6 +89,20 @@ test('don\'t display test title if there is only one anonymous test', function (
 	t.end();
 });
 
+test('known failure test', function (t) {
+	var reporter = createReporter();
+
+	var actualOutput = reporter.test({
+		title: 'known failure',
+		failing: true
+	}, createRunStatus());
+
+	var expectedOutput = '  ' + chalk.red(figures.tick) + ' ' + chalk.red('known failure');
+
+	t.is(actualOutput, expectedOutput);
```

Comment:
Why this assertion failed..
<img width="690" alt="screen shot 2016-05-23 at 15 49 48 1" src="https://cloud.githubusercontent.com/assets/3320660/15463772/05357b2a-20fe-11e6-98a8-31d00d5bb281.png">